lib/batch.js

```diff
@@ -154,18 +161,15 @@ internals.batch = function (batchRequest, resultsData, pos, parts, callback) {
                         path += value;
                     }
                     else {
-                        error = new Error('Reference value includes illegal characters');
```

Comment:
Lets not change the error handling here to use try/catch. It really should only ever be used if parsing JSON.